pkg/node/manager/manager.go

```diff
@@ -273,6 +294,19 @@ func (m *Manager) backgroundSync() {
 	}
 }
 
+// legacyNodeIpBehavior returns true if the agent is still running in legacy
+// mode regarding node IPs
+func (m *Manager) legacyNodeIpBehavior() bool {
+	// Cilium < 1.8 only exposed the Cilium internalIP to the ipcache
```

Comment:
Is this intended to be ..?

```suggestion
	// Cilium < 1.7 only exposed the Cilium internalIP to the ipcache
```

(I note this is nominated for backport to v1.7 branch).